src/tasks/task6.js

```diff
@@ -0,0 +1,9 @@
+function splicer(arr, startIndex, deleteCount, valueToInsert) {
+   if (!Array.isArray(arr)){
+      return
+   }
+   Arr.splice(0, deleteCount, valueToInsert);

```

Comment:
your function receive `arr` param, but you apply the `splice` method to `Arr`, which was not defined. fix it